### ISSUE TITLE
Skipping prohibited unicode instead of erroring

### ISSUE THREAD

**Issue by lokxii** [dankamongmen/notcurses#2773]
Currently printing string that contains prohibited unicode will result an error and stop printing (the error log pointed me to [this line](https://github.com/dankamongmen/notcurses/blob/adc7313a4d786e510df24fb9ab2108a1cd65094b/src/lib/egcpool.h#L130)). Would it be better to skip the character and logwarn (or other level of logging) that the function skipped it?

**Comment by dankamongmen** [comment]
yeah, i've thought about this. the problem is that it's really hard to indicate to the caller that there were elided glyphs, or where they were. perhaps an option, though...?

**Comment by lokxii** [comment]
The problem with stopping at prohibited unicode is that I will have to sanitize the string myself before passing to notcurses, which would duplicate the utf8 checks.

Maybe an option would be good. Maybe you can also consider putting all the alignment and styling options in an option struct? This would be a breaking change tho. Just a random suggestion.

**Comment by dankamongmen** [comment]
yep, something like that. i'm pondering what to do. open to suggestions!

**Comment by dankamongmen** [comment]
i feel this would be a toplevel option rather than a per-call deal.
